preciceAdapterFunctionObject.C

```diff
@@ -65,31 +66,70 @@ Foam::functionObjects::preciceAdapterFunctionObject::preciceAdapterFunctionObjec
 
 Foam::functionObjects::preciceAdapterFunctionObject::~preciceAdapterFunctionObject()
 {
+    #ifdef ADAPTER_TIMING_MODE
+    Info << "-------------------- preCICE adapter timers (primary rank) --------------------------" << nl;
+    Info << "Total time in adapter + preCICE: " << timeInAll_.str() << " (format: day-hh:mm:ss.ms)" << nl;
+    Info << "  For setting up (S):            " << timeInSetup_.str() << " (read() function)" << nl;
+    Info << "  For iterations (I):            " << timeInExecute_.str() << " (execute() and adjustTimeStep() functions)" << nl << nl;
```

Comment:
Is it a cumulative time or a per iteration time?

Reply:
```suggestion
    Info << "  For all iterations (I):            " << timeInExecute_.str() << " (execute() and adjustTimeStep() functions)" << nl << nl;
```
or maybe remove `For iterations` and `For setting up` completely as the categories make as cumulative quantities no real difference. Or maybe we select a different word than `iterations` such as `time loop`, which is in itself already cumulative.